core-web/libs/sdk/client/src/lib/sdk-js-client.ts

```diff
@@ -0,0 +1,121 @@
+type ClientConfig = {
+    host: string;
+    siteId: string;
+    authToken: string;
+};
+
+type PageApiOptions = {
+    path: string;
+    host_id?: string;
+    language_id?: number;
+    'com.dotmarketing.persona.id'?: string;
+    fireRules?: boolean;
+    depth?: 0 | 1 | 2;
+};
```

Comment:
should this be a class so we can encapsulate that `'com.dotmarketing.persona.id'`?

Reply:
Done